exo-run/src/service-runner.ls

```diff
@@ -18,7 +18,8 @@ require! {
 class ServiceRunner extends EventEmitter
 
   ({@name, @config, @logger}) ->
-    @service-config = yaml.safe-load (DockerHelper.get-config @config.image)
+    | @config.image  =>  @service-config = yaml.safe-load DockerHelper.get-config(@config.image)
```

Comment:
You can dry this up to:

```ls
@service-config = yaml.safe-load switch
  | @config.image  =>  DockerHelper.get-config(@config.image)
  | otherwise      =>  @fs.read-file-sync(path.join @config.root, 'service.yml')
```

Further refactoring would be to extract a method that returns the configuration for this service:

```ls
@service-config = yaml.safe-load @_service-configuration-content!

# Returns the content of the service's configuration file in textual form
_service-configuration-content: (config) ->
  | @config.image  =>  DockerHelper.get-config(@config.image)
  | otherwise      =>  @fs.read-file-sync path.join(@config.root, 'service.yml')